server/src/test/java/com/google/android/attestation/ParsedAttestationRecordTest.java

```diff
@@ -90,8 +115,9 @@ private static X509Certificate getAttestationRecord(String certStr) throws Certi
   @Test
   public void testParseAttestationRecord() throws CertificateException, IOException {
     X509Certificate x509Certificate = getAttestationRecord(CERT);
+    X509Certificate x509Certificate2 = getAttestationRecord(CERT2);
```

Comment:
Could you remove cert.checkValidity() for getAttestationRecord()? It only checks if the certificate is expired and since these are just test values I don't think it matters in this case.